### ISSUE TITLE
Issue with Typescript project

### ISSUE THREAD

**Issue by lambda0xff** [vuejs/vue-jest#245]
Issue is described here:
https://github.com/nuxt/create-nuxt-app/issues/546

After a little digging, the problem is due to `vue-jest@4.0.0-beta.3`
If I replace it with `vue-jest@4.0.0-beta.2`, the issue disappears

**Comment by nogic1008** [comment]
Error occurs here: https://github.com/vuejs/vue-jest/blob/master/lib/utils.js#L69

In `ts-jest@25.5.0` and above, `ConfigSet.typescript` property has been renamed to `parsedTsConfig`. (BREAKING CHANGE)
Ref: https://github.com/kulshekhar/ts-jest/commit/720c8546578bdbb3d6525581fbeffce82016dba5

However, this package specifies `^25.2.1`, which includes `v25.5` and later.
**`ts-jest` DOES NOT use [SemVer](https://semver.org/) for versioning.**
Therefore, version specification should be careful.

## Workaround
Install `ts-jest@25.4.0` explicitly
```bash
npm i -D vue-jest@4.0.0-beta.3 ts-jest@25.4.0
```

- [workaround sample](https://github.com/nogic1008/nuxt-ts-template/pull/192/)

**Comment by nogic1008** [comment]
To solve this issue, we can take one of three approaches:

1. Specify the version to use less than `25.5`.
    - package.json
      ```diff
      -    "ts-jest": "^25.2.1"
      +    "ts-jest": ">=25.2.1 <25.5"
      ```
1. bump `ts-jest` to `25.5.x` in [package.json](https://github.com/vuejs/vue-jest/blob/master/package.json#L76), and fix [lib/utils.js](https://github.com/vuejs/vue-jest/blob/master/lib/utils.js#L69).
    - package.json
      ```diff
      -    "ts-jest": "^25.2.1"
      +    "ts-jest": "25.5.x"
      ```
    - lib/utils.js
      ```diff
      -  const { typescript } = tr.configsFor(config)
      +  const { parsedTsConfig } = tr.configsFor(config)
      ```
1. fix [lib/utils.js](https://github.com/vuejs/vue-jest/blob/master/lib/utils.js#L69) to read `parsedTsConfig` or `typescript` property.
    - lib/utils.js
      ```diff
      -  const { typescript } = tr.configsFor(config)
      +  const configSet = tr.configsFor(config)
      +  const typescript = configSet.parsedTsConfig || configSet.typescript
      ```
----
What should I do?

**Comment by dawoe** [comment]
I was running into this issue as well... tried workaround one and that did fix the issue

**Comment by altusgerona** [comment]
I think sticking with version 3 is the safest bet. Would love to upgrade after beta is over.

**Comment by nogic1008** [comment]
solved by #246

**Comment by nogic1008** [comment]
@lambda0xff or maintainer
plz close this issue
